waiter/integration/waiter/work_stealing_integration_test.clj

```diff
@@ -71,25 +71,13 @@
             (is (<= num-instances (count responses-map))))
           (is (every? (fn [[_ num-requests]] (pos? num-requests)) responses-map)
               (str "Response distribution:" responses-map)))
+
+        ;; check slot metrics
+        (let [metrics-routers (keys (get-in service-settings [:metrics :routers]))]
+          (doseq [router metrics-routers]
+          (let [slots-in-use (get-in service-settings [:metrics :routers router :counters :instance-counts :slots-in-use])]
+            (is (zero? slots-in-use) (str "Expected zero slots-in-use, but found " slots-in-use " in router " (name router))))))
+
         ;; cleanup
         (log/info "Deleting" service-id)
-        (delete-service waiter-url service-id)))))
-
-(deftest ^:parallel ^:integration-slow test-slots-in-use-consistency
```

Comment:
Why did we delete this test?

Reply:
I combined it with the previous test.  Don't we get essentially the same coverage by checking slot consistency in the other work-stealing test?

Reply:
okay